### PR TITLE
Revert "Improve import traceback"

### PR DESCRIPTION
**Pull request by tisto**
Reverts plone/plone.restapi#1009

### CODE REVIEW

**Comment by tisto** [comment]
@jenkins-plone-org please run jobs

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

File: plone-5.1.x.cfg [hunk 1]
```diff
@@ -15,10 +15,12 @@ plone.schema = 1.2.1
 # Last pyrsistent version that is python 2 compatible:
 pyrsistent = 0.15.7
 
+# codeanalisys
+typing = 3.7.4
```
Comment (wesleybl) [inline plone-5.1.x.cfg:19]: @sneridagh I think the correct thing here is to pin `importlib-metadata = 2.1.1`. `importlib-metadata >= 3.0.0` is not compatible with Python 2:

https://github.com/python/importlib_metadata/blob/530f5da8dd3a255f1198f29ea0126b8b25e644d8/setup.cfg#L14

**Comment by wesleybl** [comment]
By the way @tisto, I think the master is not broken because of #1009. I think it's an intermittent error. If I have time I can take a look.

File: plone-5.1.x.cfg (hunk 1 above)
Comment (sneridagh) [reply plone-5.1.x.cfg:19]: thanks! I'll try, this build is killing me

**Comment by sneridagh** [comment]
@wesleybl it would be great. We need to track down this heisenbug. 
/cc @jensens It would be great if we can also pinpoint what happened in master and why it's red.

**Comment by jensens** [comment]
Well, a `__traceback_info__`` breaks nothing - except if we have some crazy doctest checking the some traceback outputs word by word.
That said, the PR tests been green before.

**Comment by sneridagh** [comment]
@jensens yeah, I know... but we need to find out why failed on master. Any clue given the traceback?

**Comment by tisto** [comment]
@wesleybl fixed master with his PR.

**Comment by wesleybl** [comment]
In fact, it is an intermittent error. I will still investigate what caused the error. But at least the master is green now.

**Pull request closed without merging**